libafl/src/executors/inprocess.rs

```diff
@@ -1608,10 +1608,10 @@ where
                     // we can't do this from the parent, timerid is unique to each process.
                     libc::timer_create(libc::CLOCK_MONOTONIC, null_mut(), addr_of_mut!(timerid));
 
-                    println!("Set timer! {:#?} {timerid:#?}", self.itimerspec);
```

Comment:
I forgot deleting this 😀 thank you

Reply:
No trouble haha, I was just getting spammed to oblivion :grin: